charts/osm/templates/osm-configmap.yaml

```diff
@@ -10,6 +10,7 @@ data:
   egress: {{ .Values.OpenServiceMesh.enableEgress | quote }}
   envoy_log_level: {{ .Values.OpenServiceMesh.envoyLogLevel | quote }}
   envoy_image: {{ .Values.OpenServiceMesh.sidecarImage | quote }}
+  init_container_image: "{{ .Values.OpenServiceMesh.image.registry }}/init:{{ .Values.OpenServiceMesh.image.tag }}"
```

Comment:
@jaellio can you hold this PR for a while? I have a PR to remove this file completely.

Reply:
Ok sure. Please go ahead